### DISCUSSION TITLE
Представления таблиц в базе данных

### DISCUSSION THREAD

**Discussion by xleonx90** [totumonline/totum-ru-issues-and-discussions#584] [Как это сделать?] [answered]
Для некоторых системных таблиц в базе данных есть "Представления" например "users__v". Есть ли возможность каким либо образом сделать представления для пользовательских таблиц. (Цель: для простых таблиц получить при запросе из базы данных поля не в формате JSON)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Конешно вы можете сделать вью средствами базы данных или запрашивать данные из ключа. Вот здесь показано как: https://ru.docs.totum.online/connect_tableau

Вью в psql создается как обычно:

```
CREATE VIEW my_view AS
SELECT 
    first_field->>'v' AS "field 1", 
    second_field->>'v' AS "field 2"
FROM 
    totum.tableau;
```

**Comment by xleonx90** [comment]
Спасибо!
